lib/vendor/PEAR/Config.php

```diff
@@ -607,7 +607,7 @@ function __construct($user_file = '', $system_file = '', $ftp_file = false,
         $this->_registry['default'] = new PEAR_Registry($this->configuration['default']['php_dir']);
         $this->_registry['default']->setConfig($this);
         $this->_regInitialized['default'] = false;
-        //$GLOBALS['_PEAR_Config_instance'] = &$this;
+        //$GLOBALS['_PEAR_Config_instance'] = $this;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)